npm problem

I can not start the main.js cause of there is problem with the tracemap.

`npm WARN TraceMap@1.0.0 No repository field.`

> root@web01:/home/users/netfreak/www/traceroute# npm start
> TraceMap@1.0.0 start /home/users/netfreak/www/traceroute
> electron
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! TraceMap@1.0.0 start: `electron .`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the TraceMap@1.0.0 start script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /root/.npm/_logs/2017-09-09T12_57_47_565Z-debug.log


Inside the log file:
> 0 info it worked if it ends with ok
> 1 verbose cli [ '/usr/bin/nodejs', '/usr/bin/npm', 'start' ]
> 2 info using npm@5.3.0
> 3 info using node@v8.4.0
> 4 verbose run-script [ 'prestart', 'start', 'poststart' ]
> 5 info lifecycle TraceMap@1.0.0~prestart: TraceMap@1.0.0
> 6 info lifecycle TraceMap@1.0.0~start: TraceMap@1.0.0
> 7 verbose lifecycle TraceMap@1.0.0~start: unsafe-perm in lifecycle true
> 8 verbose lifecycle TraceMap@1.0.0~start: PATH: /usr/lib/node_modules/npm/bin/node-gyp-bin:/home> > /users/netfreak/www/traceroute/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
> 9 verbose lifecycle TraceMap@1.0.0~start: CWD: /home/users/netfreak/www/traceroute
> 10 silly lifecycle TraceMap@1.0.0~start: Args: [ '-c', 'electron .' ]
> 11 silly lifecycle TraceMap@1.0.0~start: Returned: code: 1  signal: null
> 12 info lifecycle TraceMap@1.0.0~start: Failed to exec start script
> 13 verbose stack Error: TraceMap@1.0.0 start: `electron .`
> 13 verbose stack Exit status 1
> 13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/lib/util/lifecycle.js:289:16)
> 13 verbose stack     at emitTwo (events.js:125:13)
> 13 verbose stack     at EventEmitter.emit (events.js:213:7)
> 13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/lib/utils/spawn.js:40:14)
> 13 verbose stack     at emitTwo (events.js:125:13)
> 13 verbose stack     at ChildProcess.emit (events.js:213:7)
> 13 verbose stack     at maybeClose (internal/child_process.js:927:16)
> 13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
> 14 verbose pkgid TraceMap@1.0.0
> 15 verbose cwd /home/users/netfreak/www/traceroute
> 16 verbose Linux 4.9.0-3-amd64
> 17 verbose argv "/usr/bin/nodejs" "/usr/bin/npm" "start"
> 18 verbose node v8.4.0
> 19 verbose npm  v5.3.0
> 20 error code ELIFECYCLE
> 21 error errno 1
> 22 error TraceMap@1.0.0 start: `electron .`
> 22 error Exit status 1
> 23 error Failed at the TraceMap@1.0.0 start script.
> 23 error This is probably not a problem with npm. There is likely additional logging output above.
> 24 verbose exit [ 1, true ]

--- a/package.json
+++ b/package.json
@@ -10,6 +10,10 @@
   "build": {
     "appId": "com.tracemap.app"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/romulomourao/tracemap"
+  },
   "author": "Romulo Mourão",
   "license": "MIT",
   "devDependencies": {

